slime/templates/deployment.yaml

```diff
@@ -0,0 +1,96 @@
+{{- if .Values.deployment.enabled -}}
+{{- $root := . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "slime.fullname" $root }}
+{{- with .Values.deployment.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+  labels:
+    app: {{ template "slime.name" $root }}
+    chart: {{ template "slime.chart" $root }}
+    release: {{ $root.Release.Name }}
+    heritage: {{ $root.Release.Service }}
+{{- with .Values.deployment.labels }}
+{{ toYaml .| indent 4 }}
+{{- end }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "slime.name" $root }}
+      release: {{ $root.Release.Name | quote }}
+{{- with .Values.deployment.strategy }}
+  strategy:
+{{ toYaml . | indent 4 }}
+{{- end }}
+  revisionHistoryLimit: {{ .Values.deployment.revisionHistoryLimit }}
+  template:
+    metadata:
+      name: {{ include "slime.fullname" $root }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

Comment:
I think this should be defined in `values.yaml`.

```
annotation:
{{ .Values.deployment.metadata.annotation | tpl | indent 8 }}
```
If you need to use the helm function, use  `tpl`.

Reply:
Same as https://github.com/chatwork/charts/pull/64#discussion_r275184821, just copy monochart...😉